### PR TITLE
Os creating new button package

### PR DESCRIPTION
**Pull request by slee1130**
### Purpose 🚀
- DialogAction component
- Added a common StoryHeader component
- Changed some Readme

### Notes ✏️
_details of code change / secondary purposes of this PR_
-DialogAction.js, DialogAction.stories.js, DialogAction.spec.js, and README are added. 
-There is no styling for DialogAction so gave button space for &nbsp for now but if other tips are provided, it would be appreciated. 
@oscarkwan 

### Updates 📦
Adding new component `DialogActions`
- [x] PATCH (big fix) change to _Toast, L10n_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/OS-creating_new_button_package

### Screenshots 📸
_optional but highly recommended_
Screen Shot 2019-11-22 at 2 37 09 PM

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/DialogActions/src/DialogActions.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const defaultProps = {
+  className: "",
+  hasCancel: true,
+  hasConfirm: true,
+  hasDecline: true,
+  isDestructive: false,
+  isDisabled: false,
+  labelCancel: "Cancel", // TODO: i18n
+  labelConfirm: "Save", // TODO: i18n
+  labelDecline: "Don't Save", // TODO: i18n
```
Comment (oscarkwan) [inline packages/DialogActions/src/DialogActions.js]: Let's update this to use the `I18n`. You can find out how to do this by looking up areas in the code where it uses `I18n.t`

File: packages/DialogActions/src/DialogActions.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  onCancel: () => {},
+  onConfirm: () => {},
+  onDecline: () => {},
+};
+
+function renderCancelButton(hasCancel, onCancel, labelCancel) {
+  if (!hasCancel) return null;
+  return (
+    <Button data-pka-anchor="aclui-dialog-actions__cancel" size="large" kind="minor" onClick={onCancel}>
```
Comment (oscarkwan) [inline packages/DialogActions/src/DialogActions.js]: Any mention of `acl-ui` is not needed anymore, you can remove this everywhere else you added it as well.

Comment (oscarkwan) [reply packages/DialogActions/src/DialogActions.js]: Also you can refer to this https://github.com/acl-services/paprika/wiki/Coding-Conventions#anchors to find out how to write proper anchors with the correct conventions.

File: packages/DialogActions/test/DialogActions.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    expect(queryByText(/Save/i)).not.toBeInTheDocument();
+    expect(queryByText(/Don't Save/i)).not.toBeInTheDocument();
+    expect(queryByText(/Cancel/i)).not.toBeInTheDocument();
+  });
+
+  it("should fire callbacks when clicked", () => {
+    const onCancel = jest.fn();
+    const onConfirm = jest.fn();
+    const onDecline = jest.fn();
+    const { getByText } = renderComponent({ onCancel, onConfirm, onDecline, labelConfirm: "Confirm" });
```
Comment (oscarkwan) [inline packages/DialogActions/test/DialogActions.spec.js]: I don't think you need to pass in `labelConfirm` here.

File: packages/DialogActions/src/DialogActions.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  return (
+    <div data-pka-anchor={`aclui-dialog-actions ${props.className}`}>
+      {renderConfirmButton(
+        props.hasConfirm,
+        props.isDisabled,
+        props.onConfirm,
+        props.isDestructive,
+        props.labelConfirm
+      )}
+      &nbsp;
```
Comment (oscarkwan) [inline packages/DialogActions/src/DialogActions.js]: Don't think we need this here.. Are you trying to create margin?

File: packages/DialogActions/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+`> npm install --save @paprika/dialog-actions`
+or
+`> yarn add @paprika/dialog-actions`
+
+### Usage
+
+```js
+import DialogActions from "@paprika/dialog-actions";
+
+<>
```
Comment (oscarkwan) [inline packages/DialogActions/README.md]: You forgot this part

**Comment by oscarkwan** [comment]
@mikrotron @nahumzs Additional 👀 please 🙇

File: packages/DialogActions/src/DialogActions.js (hunk 2 above)
Comment (slee1130) [reply packages/DialogActions/src/DialogActions.js]: done

File: packages/DialogActions/src/DialogActions.js (hunk 4 above)
Comment (slee1130) [reply packages/DialogActions/src/DialogActions.js]: done

File: packages/DialogActions/README.md (hunk 5 above)
Comment (slee1130) [reply packages/DialogActions/README.md]: done

File: packages/DialogActions/test/DialogActions.spec.js (hunk 3 above)
Comment (slee1130) [reply packages/DialogActions/test/DialogActions.spec.js]: done

File: packages/DialogActions/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/icon": "^0.2.5",
+    "@paprika/raw-button": "^0.2.2",
+    "@paprika/stylers": "^0.2.2",
+    "@paprika/tokens": "^0.1.9",
+    "class-names": "^1.0.0",
```
Comment (nahumzs) [inline packages/DialogActions/package.json]: ohhh we are not using `class-name` anymore, just remove this one from the `package.json`

File: packages/DialogActions/src/DialogActions.js [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+      onClick={onDecline}
+      size="large"
+      css={ButtonStyles}
+    >
+      {labelDecline}
+    </Button>
+  );
+}
+
+function DialogActions(props) {
```
Comment (nahumzs) [inline packages/DialogActions/src/DialogActions.js]: Hi @slee1130 thank you for helping us with paprika. 

@oscarkwan did we think about using something like the following API

```jsx
<DialogActions isDisable>
   <DialogActions.Accept isDestructive onClick={() => {}} />
   <DialogActions.Cancel onClick={() => {}} />
   <DialogActions.Decline onClick={() => {}} />
</DialogActions>

```

I think this should be relative simple to implement and then we can remove complexity for our consumers. what do you think?

Comment (oscarkwan) [reply packages/DialogActions/src/DialogActions.js]: Wouldn't that be same as using
`<Button>` in the children 🤔

Comment (nahumzs) [reply packages/DialogActions/src/DialogActions.js]: Talking offline we came with a new solution :) iterating over the current API.

File: packages/DialogActions/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+```js
+import DialogActions from "@paprika/dialog-actions";
+
+<DialogActions />;
+```
+
+### Props
+
+- `className`
```
Comment (nahumzs) [inline packages/DialogActions/README.md]: we should remove className, this should be spread via `{...moreProps}` cc @oscarkwan

File: packages/DialogActions/src/DialogActions.js [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Button from "../../Button/src/Button";
+import ButtonStyles from "./DialogActions.styles";
+
+const propTypes = {
+  /** Optional custom classes. */
+  className: PropTypes.string,
+
+  /** If the primary button is a destructive action. */
+  confirmKind: PropTypes.string,
```
Comment (nahumzs) [inline packages/DialogActions/src/DialogActions.js]: shouldn't this be `kindConfirm`? following kind of the same of structure like defaultValue,

File: packages/DialogActions/src/DialogActions.js (hunk 1 above)
Comment (slee1130) [reply packages/DialogActions/src/DialogActions.js]: done

**Comment by oscarkwan** [comment]
@slee1130 i see . you comittted a package-lock.json. This happens if you run `npm` instead of `yarn`. We don't want to include this file. Can you remove it please :)

File: package.json [hunk 10]
```diff
@@ -106,6 +106,7 @@
   },
   "dependencies": {
     "@babel/runtime-corejs2": "^7.4.4",
+    "@paprika/l10n": "^0.3.2",
```
Comment (oscarkwan) [inline package.json]: can this be a `peerDependency` instead?

Comment (oscarkwan) [reply package.json]: Also this should not be in the root package.json folder. Please remove this

Comment (oscarkwan) [inline package.json]: why is this needed? @slee1130

File: packages/DialogActions/README.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+### Installation
+
+`> npm install --save @paprika/dialog-actions`
+or
+`> yarn add @paprika/dialog-actions`
+
+### Usage
+
+Please use `<L10n />` component to wrap `<Toast />` in your application.
```
Comment (nahumzs) [inline packages/DialogActions/README.md]: component to wrap `<Toast />`  🤔 ?
